index.js

```diff
@@ -143,8 +144,10 @@ module.exports = function (options = {}) {
             if (err) {
               return reject(err)
             }
+            if (matches == null || matches[0] == undefined) resolve([])
```

Comment:
No newline before if, also I consider single line if statements without `{}` as unreadable in most cases